src/pydvl/utils/parallel/backends/__init__.py

```diff
@@ -0,0 +1,9 @@
+try:
+    from .joblib import *
```

Comment:
Are there plans to make joblib optional? Since it's the default then we should always have it as a requirement and we shouldn't have this try except block.

Reply:
That's me not thinking clearly. You are correct, of course.